airflow/config_templates/airflow_local_settings.py

```diff
@@ -171,6 +171,9 @@
     },
 }
 
+# Task log shipping from Airflow components like scheduler, executor, etc.
+TASK_LOG_SHIPPER_ENABLED = conf.getboolean("logging", "enable_task_log_shipper")
```

Comment:
I know there's been some naming discussion already, but just to add my thoughts: ship/shipper is quite colloquial and might be jargon that folks in different contexts wouldn't understand. Maybe something like "transport", "export", "dispatch" (and they're related "...er" versions) might be a little more professional and clear?

Reply:
yes, after rethinking and discussing with @dstandish , it feels like all these verb indicate they are shipping/dispatching/exporting task logs to some service, but in fact, what we're doing here is sending logs to the task's context.

Hence, refactored the class from TaskLogShipper to TaskContextLogger, so it means it logs to the task's context. Hope this renaming is received better.